deployment/constructor_params/lynx/infraction.yml

```
@@ -0,0 +1,19 @@
+deployment:
+  name: infraction
+  chain_id: 80002
+
+artifacts:
+  dir: ./deployment/artifacts/
+  filename: infraction.json
+
+constants:
+  # See deployment/artifacts/lynx.json
+  COORDINATOR_PROXY: "0xE9e94499bB0f67b9DBD75506ec1735486DE57770"
+  TACO_CHILD_PROXY: "0x42F30AEc1A36995eEFaf9536Eb62BD751F982D32"
+
+contracts:
+  - InfractionCollector:
+      proxy:
+      constructor:
+          _coordinator: $COORDINATOR_PROXY
+          _tacoChildApplication: $TACO_CHILD_PROXY
```

2-space tabbing for yml file. May have originally been my fault with my original code suggestion 😅 

```suggestion
      constructor:
        _coordinator: $COORDINATOR_PROXY
        _tacoChildApplication: $TACO_CHILD_PROXY
```